global_install_scripts/update_talisman.bash

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
How is this file different from the basic installation script ?

Reply:
The install script downloads and sets up the talisman binary and hook script. It also sets up hooks in each git repository.

The update script only downloads the talisman binary and replaces it in the .talisman folder in the user home.